src/utils/supportedLanguages.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "langCode": "en",
+    "languageName": "English"
+  },
+  {
+    "langCode": "es",
+    "languageName": "Spanish"
+  },
+  {
+    "langCode": "fr",
+    "languageName": "French"
+  },
+  {
+    "langCode": "it",
+    "languageName": "Italia"
+  },
+  {
+    "langCode": "pt-br",
+    "languageName": "Brazilian Portuguese"
+  },
+  {
+    "langCode": "de",
+    "languageName": "German"
+  },
+  {
+    "langCode": "sv",
+    "languageName": "Swedish"
+  }
```

Comment:
If you don't want to translate the language name in every language later on, I recommend using the native language name here as I did to the Treecounter app here (without Swedish): https://github.com/Plant-for-the-Planet-org/treecounter-app/blob/175f7421d5aeb51a6adeabf0b263c720bbb361f3/app/components/Menu/index.js#L91-L98